platform/darwin/test/MGLRendererConfigurationTests.mm

```diff
@@ -93,4 +93,18 @@ - (void)testOverridingMGLCollisionBehaviorPre40 {
     }
 }
 
+- (void)testDefaultLocalFontFamilyName{
```

Comment:
Thanks for adding this test! Please also add tests like `testSettingMGLCollisionBehaviorPre40PListValueUsingString` and `testOverridingMGLCollisionBehaviorPre40`, which check that setting the the plist/user defaults key actually updates the config.

Reply:
@friedbunny FYI, we do not support developers setting user defaults to updates this config now. Do you think it's necessary to let developers can make that setting?